packages/flutter/lib/src/utils/parse_live_list.dart

```diff
@@ -284,7 +283,6 @@ class _ParseLiveListElementWidgetState<T extends sdk.ParseObject>
       sizeFactor: widget.sizeFactor,
       child: AnimatedSize(
         duration: widget.duration,
-        vsync: this,
```

Comment:
I'll have to test, if this really does not break the animation.

Reply:
It works, but I reverted it to avoid a breaking change when using older flutter versions.